source/includes/options-mongostat.yaml

```diff
@@ -364,4 +364,19 @@ post: |
   See :ref:`ex-mongostat-specify-columns` for an example of
   {{role}}.
 optional: true
+---
+program: mongostat
+name: interactive
+directive: option
+args: null
+pre: |
+  .. versionadded:: 3.4
+description: |
+  Display {{program}} output in an interactive non-scrolling interface
+  rather than the default scrolling output.
+  
+  {{role}} is not available with the :option:`--json <mongostat --json>`
+  option.
+  post: |
```

Comment:
Does "post: |" need to be dedented? Not sure if it's github's rendering or an issue.